### ISSUE TITLE
Issues with vlock and i3lock - neither work

### ISSUE THREAD

**Issue by xlfe** [clearlinux/distribution#1350]
Similar to #66 

@phmccarty @fenrus75 I'm unable to get either i3lock or vlock working. 

I get the following with i3lock

```
i3lock[]: pam_unix(i3lock:auth): authentication failure; logname=<myusername> uid=<myuid> euid=<myuid> tty=:0 ruser= rhost=  user=<myusername>
```

--debug on i3lock shows my password being entered correctly (I have tried > 10 times very carefully too)

vlock shows

```
Authentication service cannot retrieve authentication info.
```

Happy to perform more debugging if someone can point me at some instructions

**Comment by ahkok** [comment]
@xlfe is the user ID you are logging in `1000` or higher?

`auth     required       pam_succeed_if.so uid >= 1000 quiet`

**Comment by xlfe** [comment]
@ahkok yes it is uid 1000

**Comment by ahkok** [comment]
Thanks for replying. No idea and it'll be a while before we can get to this, sorry.

**Comment by phmccarty** [comment]
I was unable to reproduce the issue.

Perhaps you could try setting `debug` options for the pam modules installed on your system and examine that debug output in the journal? Hopefully the debug output will log more relevant details to lead to a root cause.

**Comment by xlfe** [comment]
Must just be me. I'll do some more digging / investigation and reopen the issue if there's anything that's not related just to my system.

thanks @phmccarty and @ahkok

**Comment by sunnyflunk** [comment]
It's exactly the same issue as https://community.clearlinux.org/t/issue-while-locking-screen-when-using-kde/1907/7

`unix_chkpwd` is required for `i3lock` to work. I tested `sudo mv /usr/bin/unix_chkpwd /usr/bin/unix_chkpwd_bak` which was enough to stop the screen unlocking anymore. Your issue is that you likely don't have it installed. So `Linux-PAM` is needed in the bundle/requires

**Comment by xlfe** [comment]
@phmccarty @ahkok could we add Linux-PAM (which is installed in sddm bundle) to the desktop-i3 bundle?

**Comment by xlfe** [comment]
(installing sddm fixes my vlock/i3lock problem)

**Comment by xlfe** [comment]
Thanks @sunnyflunk

**Comment by ahkok** [comment]
I'm surprised it's not yet in there to begin with, yes, we can add it there.

**Comment by ahkok** [comment]
Let me rephrase that - I'm surprised it's not pulled in through dependencies *at all*.

**Comment by ahkok** [comment]
We solved this through introducing internal dependencies on `Linux-PAM-setuid`
